### PR TITLE
Update tutorial to adhere to the Spectro Cloud style guide

### PR DESCRIPTION
**Pull request by sudkul87**
## Describe the Change

This PR removes the word `run` from the tutorial. 

## Review Changes

💻 [Add Preview URL](https://deploy-preview-1269--docs-spectrocloud.netlify.app/registries-and-packs/deploy-pack)

🎫 Jira Ticket: [DOC-724](https://spectrocloud.atlassian.net/browse/DOC-724)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1269#" title="Author self-approved">sudkul87</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by karl-cardenas-coding** [comment]
@sudkul87 let's not re-use a branch we already merged into the main branch. That's an anti-pattern and makes git history more complicated to follow and fix in the future. Create a new branch of `master` and make your changes.

File: content/docs/09-registries-and-packs/4.5-deploy-pack.md
Comment (ritawatson) [inline content/docs/09-registries-and-packs/4.5-deploy-pack.md:317]:
```suggestion
Once the registry server `/health` endpoint shows `UP` status, the next step is to log in and push the pack to it. The pack you will push is in the tutorials container. Open another bash session into the tutorials container from your local terminal.  
```

**Review by ritawatson — APPROVED** [review]
One minor suggestion - otherwise LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1269#pullrequestreview-1408213895" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1269#" title="Author self-approved">sudkul87</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request closed without merging**
